AtlasStarter/Program.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // Provide your Atlas connection string here. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
+
+            var mongoUri = "";
+
+            // The IMongoClient is the object that defines the connection to our
+            // datastore (Atlas, for example)
+            IMongoClient client;
+
+            // An IMongoCollection defines a connection to a specific MongoDB
+            // collection. Your app may have one or many different IMongoCollection
+            // objects.
+            IMongoCollection<Recipe> collection;
+
+            // Note that you must define the *type* of data stored in the
+            // IMongoCollection. We have created a class called TestDocument at
+            // the bottom of this file that serves as a "mapping class" -- the 
+            // driver maps the C# class to the BSON stored in MongoDB.
+
+            // Using mapping classes is strongly advised, but if you
+            // don't create them, you can always use the more generic BsonDocument
+            // type.
+
+            try
+            {
+                client = new MongoClient(mongoUri);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
While stack traces are helpful for experience programmers, new developers can sometimes get overwhelmed by the full trace. It might be helpful to say something like "encountered an error instantiating the mongo client (did you provide a URI with a valid username and password?) (did you whitelist your current IP)" with clear actionable items to help folks get started.